packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -320,8 +320,9 @@ export class ProposedTransformationExplorer {
                 transformByQState.getJobId(),
                 'ExportResultsArchive.zip'
             )
-
+            let exportResultsArchiveSize = 0
             let downloadErrorMessage = undefined
+            let downloadStartTime = Date.now()
```

Comment:
Use `globals.clock.Date`. Also, you will have to fix the lint suggestion to get that test to pass.